x-pack/plugins/ml/public/data_frame/common/pivot_group_by.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Dictionary } from '../../../common/types/common';
+
+export enum PIVOT_SUPPORTED_GROUP_BY_AGGS {
+  DATE_HISTOGRAM = 'date_histogram',
+  HISTOGRAM = 'histogram',
+  TERMS = 'terms',
+}
+
+type FieldName = string;
+
+interface GroupByConfigBase {
+  field: FieldName;
+  formRowLabel: string;
+}
+
+export const histogramIntervalFormatRegex = /^[0-9]+((\.)?[0-9]+)?$/;
```

Comment:
This allow a histogram interval of  `0` to be entered. You get an error displayed from the back-end, but we should probably prevent `0` being entered.

![image](https://user-images.githubusercontent.com/7405507/56726225-3ff67980-6746-11e9-9a05-13a82037d248.png)

Reply:
Fixed in e851cdd.